mule-extensions-api/src/main/java/org/mule/runtime/extension/api/declaration/type/ExtensionObjectClassTypeHandler.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.extension.api.declaration.type;
+
+import org.mule.metadata.api.builder.BaseTypeBuilder;
+import org.mule.metadata.api.builder.TypeBuilder;
+import org.mule.metadata.java.api.handler.ClassHandler;
+import org.mule.metadata.java.api.handler.TypeHandlerManager;
+import org.mule.metadata.java.api.utils.ParsingContext;
+
+import java.lang.reflect.Type;
+import java.util.List;
+
+public class ExtensionObjectClassTypeHandler implements ClassHandler {
+
+  @Override
+  public boolean handles(Class<?> clazz) {
+    return clazz.equals(Object.class);
+  }
+
+  @Override
+  public TypeBuilder<?> handleClass(Class<?> clazz, List<Type> genericTypes, TypeHandlerManager typeHandlerManager,
+                                    ParsingContext context, BaseTypeBuilder typeBuilder) {
+    return typeBuilder.anyType().id(clazz.getName());
```

Comment:
what's this for? this looks an overly simplified version of `org.mule.runtime.extension.api.declaration.type.ExtensionObjectTypeHandler` which